### ISSUE TITLE
In Admin-Ansicht: "Mitarbeiter-Status" in "Manager-Status" ändern

### ISSUE THREAD

**Issue by Zack-83** [rdmorganiser/rdmo#511]
![grafik](https://user-images.githubusercontent.com/37865804/182116451-1cd4b7c3-b431-49b8-a05b-1da976fa2bd3.png)

### Rationale / Begründung

Wenn ich die Unterscheidung richtig verstehe, können Nutzer mit "mittleren Rechten" Kataloge ändern und die Schaltfläche "Management" betreiben, aber nicht Kataloge löschen und die Schaltfläche "Admin" betreiben (das können nur diejenigen mit den höchsten Rechten).

### Minimal functionality

Ich würde "Mitarbeiter" in "Manager" editieren. Das ist kohärenter mit der Benennung der Schaltflächen und mit der vermutlichen Rolle der Person im echten Leben (FDM-Manager oder FDM-Koordinator).

Die Rollenbeschreibung könnte auch verbessert werden:

> **Aktiv**
> Legt fest, ob dieser Benutzer aktiv ist. Kann deaktiviert werden, anstatt das Konto zu löschen.
> **Manager**
> Legt fest, ob dieser Benutzer Inhalte (Fragenkataloge, Ansichte, usw.) editieren kann.
> **Administrator**
> Legt fest, ob dieser Benutzer alle Berechtigungen hat (Inhalte löschen, Rollen zuweisen, usw.), ohne diese einzeln zuweisen zu müssen.

> **Active**
> Designates whether this user should be treated as active. Unselect this instead of deleting accounts.
> **Manager**
> Designates whether this user can edit contents (catalogues, views, etc.).
> **Administrator**
> Designates whether this user has all permissions (delete contents, assign roles, etc.) without explicitly assigning them.

### Nice-to-have functionality

Wenn inaktiv, könnte ggf. die Schaltfläche "Admin" ausgeblendet werden.

**Comment by jochenklar** [comment]
Hi @Zack-83 , das ist noch ein bisschen anders, der "Mitarbeiter" status ist nichts anderes als das man sich in das Django Admin Interface einloggen kann. Normalerweise sollte das in RDMO nicht benötigt werden da "Manager" die Management Interface nutzen können.

Wenn man nicht-Admins in das Admin Interface lässt, dann kann man ihnen Rechte und/oder Gruppen zuordnen (feingranular per Model). Ich würde das aber eigentlich eher lassen.

**Comment by MyPyDavid** [comment]
ein Nutzer in der Editor Gruppe ist ein "Manager" und sieht die Management Interface?
ein Nutzer mit der Rolle Manager für die Site (site_managers) sieht alle Projekte der Site?

ich wäre auch vor eine kleine Umbenennung ;)

**Comment by jochenklar** [comment]
Ja, nur können wir das in Django nicht machen. Wir könnten das Management Interface in Content umbennen, aber das finde ich auch suboptimal. Ist ja quasi auf allen scrennshots drauf.
